app/views/landing.erb

```diff
@@ -24,17 +74,17 @@
 
     <div class="landing__text">Tuition Portal Landing Text</div>
     <div class="landing__signin">
-      <a href="/sign-in" id="sign_in_link">
+      <a href="/sign-in" id="sign_in_link" onclick="openForms(event)" name="signIn">
         <span id="sign_in_span">Sign In</span>
       </a>
     </div>
 
     <div class="landing__register">
-      <a href="/user/new" id="register_link">
-        <span id="register_span">Register</span>
+      <a href="/user/new" id="register_link" onclick="openForms(event)" name="signUp">
+        <span id="register_span" name="signUp">Register</span>
       </a>
+      <p class="message">Forgot your Password? <a href="/reset" onclick="openForms(event)" name="forgotPass">Reset your password</a></p>
```

Comment:
These `<a>` should probably lose their `href` attribute (or maybe just become `<button>` tags)... Converting them to buttons means you don't have to worry about doing `e.preventDefault()` in `openForms`